warehouse/core/src/main/java/datawave/data/hash/UID.java

```diff
@@ -43,6 +43,8 @@ public abstract class UID implements Comparable<UID>, Comparator<UID>, Writable,
 
     private static final Logger LOGGER = Logger.getLogger(UID.class);
 
+    private static final long serialVersionUID = 0L;
```

Comment:
should we be computing a more reasonable serial version UID ?  Same comment for everywhere else we use 0L

Reply:
sure!  fixed *all* the 0L, 1L, 2L, etc.